contributor-growth/drafts/contributor-growth-framework/1-Motivating.md

```diff
@@ -0,0 +1,107 @@
+   2. Workflow: Ensure the process is as intuitive and frictionless as possible.
+
+Each additional step they must take -- whether seeking info on how to contribute or submitting a PR -- represents a friction point that may demotivate them and lead to a drop-off. 
+Another less tangible but equally, if not more, important piece is the human factor. Contributors want to feel valued and belong. Creating a welcoming community is key. 
+
+## Honest and Clear Communication 
+
+The first thing prospective contributors will want to know is the level of effort it will take on their end and what they will gain -- be clear about that. This will ideally be captured in your contributor ladder (you can find a contributor ladder template here). 
+
+Shared ownership and governance is also a huge motivator. The prospect of molding the project and playing a role in its development can be a huge intrinsic motivator. Be vocal about it. When you bring in people, tell them that you're willing to share responsibility. All too often maintainers don't spell it out, assuming contributors know. This is especially important early on in the project.
+
+Be open about your roadmap and what you are trying to accomplish. Contributors want to be inspired. After all, you're asking them to dedicate their time and effort to your project. Be transparent about your vision and how you're planning on executing it. One important aspect about roadmap transparency is to let contributors know that it may change as the project matures. 
+
+When you discuss the roadmap, make sure it's all discussed publicly (e.g. community meetings, Slack, discourse, mailing list, GitHub issues or discussions). This will also help avoid situations in which someone submits a massive PR out of the blue which will almost certainly end up in a bad contributor experience. PRs aren't the best place to have feature discussion, so make sure you provide a platform for that discussion to happen (e.g Slack, GitHub Discussions, etc.). 
+
+Be clear and descriptive about how people can get involved and document it well. Good "first issues" and "help wanted" labels are great ways to signal how people can get started. It also shows that the maintainers have taken the time to identify how newcomers can help.  But also be explicit about what you need and communicate it where your users are (e.g. Twitter, Slack, tech blogs, docs, releases).  
+The more descriptive, the better. While telling contributors " just to jump in" may be easy, it can also be really overwhelming. Taking the time to describe what you need and how you need it is time-consuming, but it will also save you time down the line as people start work on things you need the way you specified them, reducing backs and forth. By describing the "what" and "how" of what you need, you are onboarding new contributors and empowering them to help other potential contributors. It's like a ponzi scheme, but for good. :)
```

Comment:
I suggest removing the ponzi scheme reference. I get the joke but I think it may distract people more than entertain?

Reply:
Yeah, that'd was deleted in suggestion mode and made it through the copy and paste. Oops!